File: src/samples/actions/testaction.pl
my $pi = 3.14159;
my $radius = 10;
for my $x (1..90) {
    $radius += 1;
    $komodo->window->moveTo(200+$radius*sin($pi*$x/12),
                            200+$radius*cos($pi*$x/12))
}

File: src/samples/actions/testaction.js
var radius = 10;
for (var x = 1; x < 90; x++) {
    radius +=1;
    komodo.window.moveTo(100+radius*Math.sin(3.14159*x/8),
                  100+radius*Math.cos(3.14159*x/8));
}